agent/grpc-external/services/peerstream/server.go

```diff
@@ -26,11 +26,12 @@ const (
 
 type Server struct {
 	Config
+
+	Tracker *Tracker
```

Comment:
Our `Config` is overloaded with what should be called dependencies; I moved Tracker out of it.